pkg/cloudprovider/providers/aws/aws.go

```diff
@@ -330,8 +330,7 @@ type Volumes interface {
 	// Check if the volume is already attached to the node with the specified NodeName
 	DiskIsAttached(diskName KubernetesVolumeID, nodeName types.NodeName) (bool, error)
 
-	// Check if a list of volumes are attached to the node with the specified NodeName
-	DisksAreAttached(diskNames []KubernetesVolumeID, nodeName types.NodeName) (map[KubernetesVolumeID]bool, error)
+	CheckClusterDisks(map[types.NodeName][]KubernetesVolumeID) (map[types.NodeName]map[KubernetesVolumeID]bool, error)
```

Comment:
hmm, so just renaming the function or event putting it back doesn't get rid of change from admission controller. if if use `DisksAreAttached` same as `CheckClusterDisks` then function signature is changed and that would still mean changing admission plugin tests.